src/components/ChatApp/Settings/ShareOnSocialMedia.js

```diff
@@ -96,14 +96,14 @@ const ShareOnSocialMedia = props => {
             />
           </div>
           <div style={buttonStyle}>
-            <Translate text="Share about SUSI on Google +" />
+            <Translate text="Share about SUSI on Google+" />
             <br />
             <RaisedButton
               label={<Translate text="Share on Google+" />}
               style={raisedButtonStyle}
               backgroundColor="#d34836"
               labelColor="#fff"
-              icon={<FontIcon className="fa fa-google-plus" />}
+              icon={<FontIcon className="fa fa-google-plus-circle" />}
```

Comment:
This icon is inconsistent with the other icons. Please revert it back to **fa-google-plus**